group_vars/gearshift_cluster/vars.yml

```diff
@@ -65,6 +65,9 @@ nameservers: [
   '8.8.4.4',       # Google DNS.
   '8.8.8.8',       # Google DNS.
 ]
+rsyslog_remote_servers:
```

Comment:
_Remote rsyslog servers_ is a bit redundant and therefore confusing: the ```r``` in rsyslog already stands for ```remote```. I think you mean rsyslog servers not managed with code from this repo and hence not listed in the inventories, right? If that is correct, how about ```rsyslog_external_servers```?